packages/client/components/ActivityLibrary/ActivityLibraryCard.tsx

```diff
@@ -1,21 +1,88 @@
-import React from 'react'
+import clsx from 'clsx'
+import React, {PropsWithChildren, ComponentPropsWithoutRef, ElementType, ReactNode} from 'react'
+import {MeetingTypeEnum} from '../../__generated__/ActivityLibraryQuery.graphql'
+import {ActivityCard} from './ActivityCard'
 
-interface Props {
-  name: string
-  type: string
-  teamName: string
+const Image = (props: PropsWithChildren<React.ImgHTMLAttributes<HTMLImageElement>>) => {
+  const {className, src} = props
+
+  return (
+    <div className='absolute inset-0 top-5 flex'>
+      <img className={clsx('m-auto h-[76px] w-auto object-contain', className)} src={src} />
+    </div>
+  )
+}
+
+interface TitleProps<T extends ElementType> {
+  as?: T
+  children: ReactNode
 }
 
-const ActivityLibraryCard = (props: Props) => {
-  const {type, name, teamName} = props
+const Title = <T extends ElementType = 'div'>(
+  props: TitleProps<T> & ComponentPropsWithoutRef<T>
+) => {
+  const {as, children, className, ...rest} = props
+  const Component = as || 'div'
+
   return (
-    <div className='m-2 border-solid p-2'>
-      <div>
-        {type} on team {teamName}
-      </div>
-      <div className='text-lg font-bold'>{name}</div>
+    <Component
+      className={clsx(
+        'z-10 block pr-6 text-sm font-semibold leading-5 text-slate-800 focus:outline-none sm:text-base',
+        className
+      )}
+      {...rest}
+    >
+      {/* Extend touch target to entire activity card if it's a link */}
+      {'to' in props && <span className='absolute inset-0' aria-hidden='true' />}
```

Comment:
`a` tags can't contain divs so this is a way of extending the touch/link area to the whole card while still maintaining all the accessibility features